src/components/ChatEntry.js

```diff
@@ -1,22 +1,46 @@
-import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const onLikeButtonClick = () => {
+    const updateChatEntry = {
+      id: props.id,
+      sender: props.sender,
+      body: props.body,
+      timeStamp: props.timeStamp,
+      liked: !props.liked,
+    };
+    props.onUpdateLiked(updateChatEntry);
+  };
+
+  const heart = props.liked ? '❤️' : '🤍';
+  const alignment = props.sender === 'Vladimir'? 'chat-entry local' : 'chat-entry remote';
+  const bubbleAlign = props.sender === 'Vladimir' ? 'entry-bubble local' : 'entry-bubble remote';
+
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
-      <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+    <div className={alignment}>
```

Comment:
Referring to my previous comment, string interpolation would look like:
```suggestion
    <div className={`chat-entry ${alignment}`}>
```